index.js

```diff
@@ -1,60 +1,57 @@
+module.exports = function createApplicationConfig (name) {
+  return new ApplicationConfig(name)
+}
```

Comment:
Could you move this to the bottom of the file? I prefer not to rely on function hoisting...